packages/turf-polygon-smooth/index.js

```diff
@@ -0,0 +1,55 @@
+import { coordEach, geomEach } from '@turf/meta';
+import { featureCollection, polygon, isObject } from '@turf/helpers';
+
+/**
+ * Smooths a {@link Polygon}. Based on [Chaikin's algorithm](http://graphics.cs.ucdavis.edu/education/CAGDNotes/Chaikins-Algorithm/Chaikins-Algorithm.html).
+ * Warning: may create degenerate polygons.
+ *
+ * @name polygonSmooth
+ * @param {FeatureCollection<Polygon>} inputPolys to smooth
+ * @param {Object} [options] Optional parameters
+ * @param {string} [options.iterations] THe number of times to smooth the polygon. A higher value means a smoother polygon.
+ * @returns {FeatureCollection<Polygon>} FeatureCollection containing the smoothed polygon/poylgons
+ * @example
+ * var polygon = turf.polygon([[[11, 0], [22, 4], [31, 0], [31, 11], [21, 15], [11, 11], [11, 0]]]);
+ *
+ * var smoothed = turf.polygonSmooth(polygon)
+ *
+ * //addToMap
+ * var addToMap = [smoothed, polygon];
+ */
+function polygonSmooth(inputPolys, options) {
+    var outPolys = [];
+    options = options || {iterations: 1};
```

Comment:
🤔 Don't think this would be the best way to define default settings, in the case of options given as an empty `Object` then the default wouldn't be defined as `1`.

```js
turf.polygonSmooth(polygon, {})
// options.iterations => undefined
```